package.json

```diff
@@ -53,7 +53,7 @@
         "@angular/platform-browser-dynamic": "^17.3.0",
         "@angular/router": "^17.3.0",
         "@egjs/hammerjs": "^2.0.17",
-        "@geoengine/openapi-client": "0.0.9",
+        "@geoengine/openapi-client": "https://gitpkg.vercel.app/1lutz/openapi-client/typescript?main",
```

Comment:
remember to change this to the new published openapi client package once it is released and before merging the ui

Reply:
05463fcb046d60f38f99b77145682a596aea5497